CalendarApp/BookingRoom/Controllers/PostEventController.cs

```diff
@@ -19,8 +22,17 @@ public PostEventController(BookingService bookingService)
         [HttpPost]
         public HttpResponseMessage EventPost(CalendarEvent eventPost)
         {
-            BookingService.PostEvent(ToEventConverter.ToEvent(eventPost), eventPost.CalendarID);
+            List<Event> conflictedEvents = BookingService.ManagerEvent.CheckEvent(eventPost.CalendarID, ToEventConverter.ToEvent(eventPost), eventPost.ForcedEvent);
```

Comment:
I'd extract ToEventConverter.ToEvent(eventPost) to a separate variable. This will make code more readable